database/queries.go

```diff
@@ -25,7 +25,9 @@ func (db *Handler) BeginEndEntityOperations() (*Transaction, error) {
 		return nil, err
 	}
 	// lock the table
-	_, err = tx.Exec("LOCK TABLE endentities_lock IN ACCESS EXCLUSIVE MODE")
+	_, err = tx.Exec(`LOCK TABLE endentities IN EXCLUSIVE MODE;
```

Comment:
This is dope. How hard would it be to feature flag this just in case we need to roll it back?

Reply:
It would just be a env var and an if statement to do the old way vs this way. Is that worth doing for this or would we rather just roll back to the previously deployed container?